loghub/main.py

```diff
@@ -164,7 +166,7 @@ def format_changelog(repo,
         version=version, date=close_date, q=quotes)
 
     lines.append(header)
-    lines.append('### Bugs fixed\n')
+    lines.append('### Issues closed\n')
```

Comment:
It's confusing to have a title that says `Issues closed` and under it two sections called `Issues` and `Pull requests`